src/lib/temple/activity-new/fetch.ts

```diff
@@ -236,4 +246,4 @@ async function fetchOperGroupsForOperations(
  * > `{"code":400,"errors":{"lastId":"The value '331626822238208' is not valid."}}`
  * > when it's not true!
  */
-const buildOlderThanParam = (olderThan?: Activity) => ({ 'timestamp.lt': olderThan?.oldestTzktOperation?.timestamp });
+const buildOlderThanParam = (olderThan?: Activity) => ({ 'timestamp.lt': olderThan?.timestamp });
```

Comment:
Sorry, the first point of the comment makes no sense here because `Activity` interface is absolutely another now and looks more like one operation. Moreover, I can't see any use of adding `'level.lt'` filter because I have never seen blocks with two different levels with same timestamps or with reverse order of them